test-framework/sudo-compliance-tests/src/visudo/flag_file.rs

```diff
@@ -174,10 +174,9 @@ echo "$@" > {LOGS_PATH}"#
 }
 
 #[test]
-#[ignore = "gh657"]
 fn regular_user_can_create_file() -> Result<()> {
     let env = Env("")
-        .file(DEFAULT_EDITOR, TextFile(EDITOR_TRUE).chmod(CHMOD_EXEC))
+        .file(DEFAULT_EDITOR, TextFile(EDITOR_TRUE).chmod("111"))
```

Comment:
is this necessary to make the test pass with sudo-rs? ogsudo was passing it so I guess it still created the file and exited with code 0 even when the editor was not executable

Reply:
Yes! I spend like 10 mins trying to figure out what happened until I checked the owner of `/usr/bin/editor` and noticed it was root. So yeah it seems visudo doesn't propagate the exec failure into the status code sometimes either :shrug: